tests/server/auth/test_auth.py

```diff
@@ -25,7 +25,7 @@
 
 @pytest.fixture
 def client(tmp_path):
-    path = tmp_path.joinpath("sqlalchemy.db").as_uri()
+    path = tmp_path.joinpath("mlruns.sqlite").as_uri()
```

Comment:
Interesting. Where does `mlruns.sqlite` come from?

Reply:
this line:
https://github.com/mlflow/mlflow/blob/5f4ae6a38a4402ab2ad3a07dfa82847417e87b9c/tests/conftest.py#L35

Reply:
Why is the run logged in `mlruns.sqlite`?

Reply:
Can we use `_get_tracking_store` instead of `get_run`?

https://github.com/mlflow/mlflow/blob/178dabfaabf8242b30130567342a3042cf286cb0/mlflow/server/handlers.py#L251

Reply:
good point, that makes sense

Reply:
@gabrielfu I filed https://github.com/mlflow/mlflow/pull/8839. We should not set `MLFLOW_TRACKING_URI` in `tracking_uri_mock`.